subscriptionform/templates/sports/participant_list.html

```diff
@@ -4,7 +4,7 @@
     <h2>Participants</h2>
     <uL>
         {% for participant in participant_list %}
-            <li>{{ participant.first_name }} {{ participant.last_name }}</li>
+            <li><a href="{% url 'participant_detail' pk=participant.pk %}">{{ participant.first_name }} {{ participant.last_name }}</a></li>
```

Comment:
je zou hier een property van kunnen maken, die full_name heet en die het model dan kent en terug kan geven.

Zie @property:
http://stackoverflow.com/questions/17682567/how-to-add-a-calculated-field-to-a-django-model